src/ubuntu/24.04/sdk/amd64/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM mcr.microsoft.com/dotnet/sdk:10.0-preview
```

Comment:
Why are we not using Azure Linux?

Reply:
Yeah, good point. Should be `10.0-preview-azurelinux3.0` instead.